src/components/Chip/Chip.tsx

```diff
@@ -51,13 +25,13 @@ export interface ChipProps {
 const Chip: React.FC<ChipProps> = ({ disabled = false, ...props }) => {
 	return (
 		<button
-			key={props.key}
+			{...props}
 			className={
-				'text-h5 color-white cursor-pointer ' +
+				'cursor-pointer font-lato text-sm text-white' +
 				`prefer-${props.variant ? props.variant : 'primary'} ${props.className} ` +
-				`disabled-${disabled} `
+				`disabled-${disabled} border-2 border-solid ${props.backgroundColor}`
```

Comment:
@manishdoley23 remove disbaled-${disabled} class, not available in tailwind, add disabled prop in div.